src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_031_NuGetApiKey.cs

```diff
@@ -36,6 +36,22 @@ public override Tuple<string, string> GetMatchIdAndName(string match)
             return base.GetMatchIdAndName(match);
         }
 
+#if HIGH_PERFORMANCE_CODEGEN
+        IEnumerable<HighPerformancePattern> IHighPerformanceScannableKey.HighPerformancePatterns => [
+            new(signature: "oy2",
+                scopedRegex: @"^.{3}[a-z2-7]{43}",
```

Comment:
>.{3} [](http://example.com/codeflow?start=32&length=4)

same question, will stop asking it, why do we not recap the actual signature here?

Reply:
resolving this one since I answered elsewhere.